plugin/src/main/kotlin/io/qalipsis/gradle/cloud/dsl/CampaignConfigurationSpecification.kt

```diff
@@ -0,0 +1,56 @@
+package io.qalipsis.gradle.cloud.dsl
+
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.Property
+
+/**
+ * Base class for defining campaign configuration specifications.
+ *
+ * @author Francisca Eze
+ */
+@CampaignMarker
+abstract class CampaignConfigurationSpecification {
+
+    abstract val campaignName: Property<String>
```

Comment:
Make it a parameter of the constructor, therefore as simple String instead of Property<String>